api/docs/debugging.dox

```diff
@@ -78,12 +78,10 @@ the shared library that identifies the debug file.
 
 ## Expected Signals
 
-On Linux, DR sends itself a SIGILL signal during initialization, in order to measure the size of the signal frame used by the kernel.  If you are under gdb, simply continue past this signal.
+On Linux, DR sends itself a SIGILL signal during initialization, in order to measure the size of the signal frame used by the kernel.  If you are under gdb, simply continue past this signal.  SIGILL (SIGFPE on MacOS; SIGSTKFLT inside QEMU) is also sent to other threads to suspend or terminate them later on during execution.
```

Comment:
Also document lack of gdb support on qemu.

Reply:
Added a mention.  There are many issues with qemu and I think using gdb on DR on QEMU is already not very workable even w/o SIGSTKFLT.